docs/release-notes/release-notes-0.14.0.md

```diff
@@ -25,6 +25,8 @@ for more information.
 * [Stub code for interacting with `lnrpc` from a WASM context through JSON 
   messages was added](https://github.com/lightningnetwork/lnd/pull/5601).
 
+* [Add `peersrpc` service with `UpdateNodeAnnouncement` endpoint](https://github.com/lightningnetwork/lnd/issues/5466).
```

Comment:
@guggero this is not done yet, I build the functionality but not migrated it to its own sub service yet